R/metab.bookkeep.R

```diff
@@ -113,7 +113,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 
 	R <- mean(nep.night, na.rm=TRUE) * freq # should be negative
 	NEP <- mean(delta.do.metab, na.rm=TRUE) * freq # can be positive or negative
-	GPP <- mean(nep.day, na.rm=TRUE) * sum(dayI) - R # should be positive
+	GPP <- mean(nep.day, na.rm = TRUE) * sum(dayI) - mean(nep.night, na.rm = TRUE) * sum(dayI) # should be positive
```

Comment:
Yeah, I confirm this is right. Otherwise it is over-calculating the total amount of respiration during the sunlight period. 

This brings up another point though, the calculation of dayI and nightI above, I think, needs fixing. Length is off by one when subsetting `delta.do.metab` on lines 110-111.